src/routes/GroupSidebar/GroupSidebar.js

```diff
@@ -4,6 +4,7 @@ import { Link } from 'react-router-dom'
 import { object, string, array } from 'prop-types'
```

Comment:
i think we're about to get rid of GroupSidebar and replace with GroupDetail, but ok to have these changes here for now